frontend/src/components/EasyModal/EasyModalHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import RightCloseButton from "../RightCloseButton";
+
+export function EasyModalHeader({
+  showCloseButton,
+  title,
+  handleClose,
+}: {
+  title: string;
+  handleClose: () => void;
+  showCloseButton?: true;
+}) {
+  return (
+    <div className="w-full h-10 justify-between items-center inline-flex">
+      <div className="p-2 text-zinc-800 text-xl font-normal leading-normal">
+        <h2>{title}</h2>
+      </div>
+      {showCloseButton && (
+        <div className="p-2 rounded-lg justify-center items-center gap-2 flex">
+          <RightCloseButton onClick={handleClose} />
+        </div>
+      )}
+    </div>
```

Comment:
Dette ligner veldig på noe jeg rettet på i forrige PR, usikker på om endringene da kanskje ikke har blitt med hit? 😅 Foreslår uansett å endre til: 

```suggestion
  <div className="flex flex-row justify-between items-center">
      <h2>{title}</h2>
      {showCloseButton && <RightCloseButton onClick={handleClose} />}
    </div>
```

Reply:
Det kan virkelig stemme ja 🤔 Fikk en lite gøyal merge-conflict isted som jeg gjorde mye rart for å løse. Så beklager dette.